src/modules/roc_fec/writer.h

```diff
@@ -55,7 +55,7 @@ class Writer : public packet::IWriter, public core::NonCopyable<> {
     bool valid() const;
 
     //! Set number of source packets per block.
-    void resize(size_t sblen);
+    bool resize(size_t sblen);
```

Comment:
We should also replace `resize(n)` with `CHECK(resize(n))` everywhere in tests.

Reply:
Done